docs/standards/lsp-background/erc725.md

```diff
@@ -14,69 +13,78 @@ sidebar_position: 1
 
 ## Introduction
 
-Using Externally Owned Accounts (EOA) is limiting and risky as if the private key of the account gets leaked, anyone can execute any transaction from it, transferring all assets owned by the account. Also, data cannot be attached to it and it's limited to performing the CALL and CREATE opcode.
-
-In order to have a more secure account, more flexible execution on the blockchain and the ability to attach endless data, a smart contract should be used. It ensures a safe ownership process, more operations to execute and endless data store availability.
-
-
-## What does this standard represent?
+EOAs (Externally Owned Accounts) are controlled by private keys and can perform specific operations such as **CALL** to interact with other addresses and **CREATE** to create other smart contracts. However, they do not have any internal logic or storage capabilities.
 
-This standard is formed from 2 sub-standards that could be implemented stand-alone or together.   
-**[ERC173](https://eips.ethereum.org/EIPS/eip-173)** is used in both standards to provide ownership on the smart contract functions, where the owner could be an Externally Owned Account (EOA) or a smart contract (Another ERC725 contract, Key Manager contract, etc).
+Smart contracts, on the other hand, can be controlled by addresses or other smart contracts, which provide a flexible ownership management mechanism. They can store data and perform more complex operations, making them useful for a wide range of use cases, including digital assets, decentralized applications, DAO and smart contract based accounts.
```

Comment:
i suggest to replace "controlled by addresses" to "controlled by EOAs" because in the paragraph above, you explain what an EOA is, not what an "address" is. So reader will be like: "Ok, EOA, I know what it is. But what is an address?" Bc it is not explained above that EOA ~~ addresses